site/content/integrate/apps/deployment/_index.md

```diff
@@ -52,40 +49,132 @@ For a go app the manifest snippet would look like this:
         }
     ]
 }
-``` 
+```
+
+### Setting up your Mattermost instance and AWS for deploying apps
+
+Note that apps are not supported in the Mattermost Cloud - they're only used for developer testing convenience. 
+
+Provisioning in the AWS cloud environment will be done by the **appsctl** tool described in the following AWS Setup section.
+
+We need an app bundle to provision an app. The bundle might be provisioned from the local disk, from S3 (not implemented yet), or from a URL (not implemented yet). Provisioning consists of three parts which will be completed via the **appsctl** tool:
+
+1. Creating the lambda functions with appropriate policies.
+2. Storing static assets in the dedicated S3 bucket.
+3. Storing the app’s manifest file in the same dedicated S3 bucket.
+
+#### Mattermost setup
+
+Follow the Prerequisites section of the [JavaScript]({{< ref "quick-start-js" >}}) or [Go]({{< ref "quick-start-go" >}}) Quick Start Guides to set up your Mattermost instance. Ensure that you install the [apps plugin](https://developers.mattermost.com/integrate/apps/quick-start-js/#install-the-apps-plugin) as part of this process.
```

Comment:
Please also migrate this URL to use the hugo ref syntax.

Reply:
done!  I check that the rest of the file uses hugo references for any references to developers.mattermost.com in this file.

Reply:
[`047951a` (#826)](https://github.com/mattermost/mattermost-developer-documentation/pull/826/commits/047951a0b1d3640b90ed7f7a23c15b4257c7472a)